.github/ISSUE_TEMPLATE/zpa_epic.md

```diff
@@ -0,0 +1,80 @@
+---
+
+name: Zeebe Process Automation team Epic
+about: Prepare a new Epic for the ZPA team
+title: ''
+labels: 'kind/epic'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
```

Comment:
Do we need this part in our EPICs?

Reply:
I copied the feature request template, forgot to take this out 😄